docs/topics/signing.txt

```diff
@@ -156,23 +182,49 @@ created within a specified period of time::
         otherwise raises ``SignatureExpired``. The ``max_age`` parameter can
         accept an integer or a :py:class:`datetime.timedelta` object.
 
+    .. method:: sign_object(obj, serializer=JSONSerializer, compress=False)
+
+        .. versionadded:: 3.2
+
+        Encode, optionally compress, sign complex data structure (e.g. list,
```

Comment:
Nitpicking, but maybe add an "add" here? "…, and sign complex data"

Reply:
There is _"and"_ before the last _", and append current ..."_.

Reply:
ha, reading comprehension skills… But then the ordering is wrong. Signing is always last (ie the timestamp is also signed!)

Reply:
Good catch :dart: , updated.